website/snippets/_trigger-a-ci-job.md

```diff
@@ -0,0 +1,2 @@
+If you're not using dbt Cloud’s native Git integration with [GitHub](/docs/cloud/git/connect-github), [GitLab](/docs/cloud/git/connect-gitlab), or [Azure DevOps](/docs/cloud/git/connect-azure-devops), you can use the [Administrative API](/docs/dbt-cloud-apis/admin-cloud-api) to trigger a CI job to run. However, dbt Cloud will not automatically delete the temporary schema for you. This is because automatic deletion relies on incoming webhooks from Git providers, which is only available through the native integrations.
```

Comment:
this is a silly question but why are you turning this into a snippet? and why isn't it be used if turning into a snippet?

Reply:
it seems that that content is part of this section: https://docs.getdbt.com/docs/deploy/ci-jobs#trigger-a-ci-job-with-the-api